tests/test_lint.py

```diff
@@ -271,4 +271,18 @@ def test_lint_lines_read_errors(fake_repository):
     assert "read error" in result
 
 
+def test_lint_specific_files(fake_repository):
+    """Check lint-file subcommand."""
+    (fake_repository / "foo.py").write_text("foo")
+    (fake_repository / "bar.py").write_text("bar")
+
+    project = Project.from_directory(fake_repository)
+    report = ProjectReport.generate(project, file_list=["foo.py"])
+    result = format_plain(report)
+
+    assert ":-(" in result
+    assert "# UNUSED LICENSES" in result
```

Comment:
This is incorrect. The purpose of `lint-file` is not to do the exact same thing as `lint`, but on a subset of the project. The purpose of `lint-file` is to do a subset of what `lint` does on a subset of the project. Or, crudely put, `lint-file` should check whether each file has a copyright holder and a licence. In addition, it wouldn't hurt to also check for missing licences. The rest is covered by `lint`.

The idea is that this makes for a quicker pre-commit hook that doesn't check absolutely everything in a project.

Reply:
Implemented this!